packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -60,9 +65,15 @@ const useIntentIconStyles = makeStyles({
  */
 export const useAlertStyles_unstable = (state: AlertState): AlertState => {
   const styles = useStyles();
+  const appearance = state.appearance || 'primary';
```

Comment:
If we make appearance required in state you wouldn't need to do this and you could just use `state.appearance` directly below.